### PR TITLE
Asset sync and kube dependency update for OpenShift 4.16 GA

### PR DESCRIPTION
**Pull request by Evan-Reilly**

### CODE REVIEW

**Review by rtheis — CHANGES_REQUESTED** [review]
Please run `make bindata` and fix the PR test failures.

**Review by rtheis — CHANGES_REQUESTED** [review]
PR tests are failing. It appears that you'll need to change more than just the Go dependencies.

```

Golang errors in file: []:
  > [line 0 : column 0] - # command-line-arguments
pkg/controllers/openshiftcontrollermanager/operator_client.go:39:9: cannot use &cmOperatorClient{…} (value of type *cmOperatorClient) as v1helpers.OperatorClient value in return statement: *cmOperatorClient does not implement v1helpers.OperatorClient (missing method GetOperatorStateWithQuorum)
  > [line 0 : column 0] - # command-line-arguments
pkg/controllers/openshiftapiserver/controller.go:103:9: cannot use &apiServerOperatorClient{…} (value of type *apiServerOperatorClient) as v1helpers.OperatorClient value in return statement: *apiServerOperatorClient does not implement v1helpers.OperatorClient (missing method GetOperatorStateWithQuorum)
Golang errors in file: [/go/src/github.com/openshift/ibm-roks-toolkit/pkg/controllers/openshiftapiserver/controller.go]:
  > [line 103 : column 9] - cannot use &(apiServerOperatorClient literal) (value of type *apiServerOperatorClient) as v1helpers.OperatorClient value in return statement: *apiServerOperatorClient does not implement v1helpers.OperatorClient (missing method GetOperatorStateWithQuorum)
Golang errors in file: [/go/src/github.com/openshift/ibm-roks-toolkit/pkg/controllers/openshiftcontrollermanager/operator_client.go]:
  > [line 39 : column 9] - cannot use &(cmOperatorClient literal) (value of type *cmOperatorClient) as v1helpers.OperatorClient value in return statement: *cmOperatorClient does not implement v1helpers.OperatorClient (missing method GetOperatorStateWithQuorum)
Summary:
  Gosec  : 2.14.0
  Files  : 34
  Lines  : 11693
  Nosec  : 7
  Issues : 0 
```

File: go.mod [hunk 1]
```diff
@@ -6,63 +6,59 @@ require (
...
-	k8s.io/apiserver v0.29.4
-	k8s.io/cli-runtime v0.29.4
-	k8s.io/client-go v0.29.4
-	k8s.io/component-base v0.29.4
+	k8s.io/api v0.29.7
+	k8s.io/apiextensions-apiserver v0.29.7
+	k8s.io/apimachinery v0.29.7
+	k8s.io/apiserver v0.29.7
+	k8s.io/cli-runtime v0.29.7
+	k8s.io/client-go v1.5.2
```
Comment (jonesbr17) [inline go.mod:31]: We want this to stay in line with the rest of the k8s deps, so it should be at `v0.29.7`

**Review by jonesbr17 — CHANGES_REQUESTED** [review]

**Review by rtheis — APPROVED** [review]
/lgtm
/approved

@jonesbr17 ptal

**Review by jonesbr17 — APPROVED** [review]
/lgtm
/approve

**Comment by rtheis** [comment]
@Evan-Reilly you need to manually cherry-pick this PR to 4.16.

**Pull request merged**